### PR TITLE
Add `-l` option to `neuro image tags` to print long output including tag size

### PR DESCRIPTION
**Pull request by anayden**
Work in progress.

### CODE REVIEW

**Comment by yartem** [comment]
Minor: I'm getting badly formatted output "Tags for <image_name>...":
image

**Comment by anayden** [comment]
Good point. Fixed the issue

**Review by asvetlov — CHANGES_REQUESTED** [review]
Please document the new API

File: docs/images_reference.rst [hunk 1]
```diff
@@ -104,6 +104,26 @@ Images
...
+
+
+   .. comethod:: tag_info(image: RemoteImage) -> Tag
+
+      Return info about specified tag.
+
+      :param RemoteImage image: a spec for remote image with tag on Neuro
+                                registry.
+
+      :return: tag information (name and size) (:class: `Tag`)
```
Comment (asvetlov) [inline docs/images_reference.rst:124]:
```suggestion
      :return: tag information (name and size) (:class:`Tag`)
```

**Review by asvetlov — APPROVED** [review]

**Pull request merged**